all/099-ceph.yml

```diff
@@ -172,6 +172,11 @@ openstack_keys_defaults:
       mon: "profile rbd"
       osd: "profile rbd pool={{ openstack_glance_pool.name }}, profile rbd pool={{ openstack_nova_pool.name }}, profile rbd pool={{ openstack_cinder_pool.name }}, profile rbd pool={{ openstack_cinder_backup_pool.name }}"
     mode: "0600"
+  - name: client.manila
+    caps:
+      mon: "profile rbd"
+      osd: "profile rbd pool={{ cephfs_data_pool.name }}"
```

Comment:
Are you sure about the "profile rbd" part? In the manila docs this looks different https://docs.openstack.org/manila/latest/admin/cephfs_driver.html#authorizing-the-driver-to-communicate-with-ceph